Documentation/HowToGuides/PassingAFloatValueFromAnInteractorToAnInteractable/README.md

```diff
@@ -0,0 +1,190 @@
+# Passing A Float From An Interactor To An Interactable
+
+> * Level: Intermediate
+>
+> * Reading Time: 10 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction 
+
+## Prerequisites
+
+* [Add the Tilia.Interactions.Interactable.Unity] prefab to the scene Hierarchy.
+
+## Let's Start
+
+### Step 1
+
+Disable `CameraRigs.SpatialSimulator` and enable `CameraRigs.UnityXR`
+
+![Disable Camera Rig](assets/images/DisableCameraRig.png)
+
+### Step 2
+
+Expand the `Tilia Input UnityInputManager` Package directory in the Unity Project window and then select `Packages -> Runtime -> Prefabs -> ControllerMappings`. For this example we are going to add the `Input.UnityInputManager.Oculus.TouchLeftController` prefab and the `Input.UnityInputManager.Oculus.TouchRightController`prefab.
+
+Drag and Drop those two items into the Unity Scene.
+
+![Expand Directory](assets/images/ExpandDirectory.png)
```

Comment:
show arrows in the drag and drop
![image](https://user-images.githubusercontent.com/1029673/74438042-2fdc9180-4e61-11ea-8214-d8901f59b9c6.png)